### PR TITLE
templates: prevent Angular to execute inline CSS

### PR DESCRIPTION
**Pull request by diegodelemos**
Signed-off-by: Diego Rodriguez <diego.rodriguez@cern.ch>

### CODE REVIEW

**Comment by drjova** [comment]
could you please rebase :P

**Comment by diegodelemos** [comment]
give me a second... I have just checked on Firefox and I am still getting some warnings.. the battle is not over 🔫

**Comment by diegodelemos** [comment]
rebased everything 👍  the warnings are for the rest of inlines we have spread all over other modules

**Comment by drjova** [comment]
thanks, can we track them down?

**Comment by diegodelemos** [comment]
Some of them we already know such as `invenio-i18n` and `invenio-search-js`, the rest should be a matter of greping around

I have tested the CSP `default-src: 'self'` and it seems that nothing is breaking, I have updated the PR to use ng-csp without parameters (son `unsafe-eval` and `unsafe-inline` are deactivated) and I will update `invenio-app` so we keep the CSP directive restrictive as before. Once it is done, could you please check it out so we make sure nothing is breaking? 🦄

File: invenio_theme/static/scss/invenio_theme/styles.scss [hunk 1]
```diff
@@ -36,3 +36,26 @@ $fa-font-path: "/static/node_modules/font-awesome/fonts";
...
+
+// CSS code to allow `ng-csp="no-inline-css"`
+[ng\:cloak],
+[ng-cloak],
+[data-ng-cloak],
+[x-ng-cloak],
+.ng-cloak,
+.x-ng-cloak,
+.ng-hide:not(.ng-hide-animate) {
+    display: none !important;
```
Comment (drjova) [inline invenio_theme/static/scss/invenio_theme/styles.scss]: could you please keep two spaces for indentation? :)

**Comment by drjova** [comment]
@diegodelemos perfect, please have a look on the comment and I will make a test drive :P

File: invenio_theme/static/scss/invenio_theme/styles.scss (hunk 1 above)
Comment (diegodelemos) [inline invenio_theme/static/scss/invenio_theme/styles.scss]: 👍

**Comment by diegodelemos** [comment]
I have included a commit adding the logic to load `i18n` assets (just for the purpose of speed and not having two PRs)

**Pull request merged**